addons/web_unsplash/static/src/components/media_dialog/image_selector.js

```diff
@@ -228,8 +228,11 @@ patch(uploadService, 'upload_service_unsplash', {
                     name: records.length > 1 ?
                     _.str.sprintf(env._t("Uploading %s '%s' images."), records.length, records[0].query) :
                     _.str.sprintf(env._t("Uploading '%s' image."), records[0].query),
-                    size: null,
+                    size: "",
                     progress: 0,
+                    hasError: false,
+                    uploaded: false,
+                    errorMessage: "",
```

Comment:
Instead of forcing to add those when calling `addFile`, should not those be the default values somehow?